app/views/votes/new.html.erb

```diff
@@ -93,67 +99,82 @@
         <div class="row justify-content-center">
           <div class="d-flex justify-content-center">
 
+
             <% if @challenge.challenger.photo.attached? %>
               <%= cl_image_tag @challenge.challenger.photo.key, class: 'vote-image', height: 300, width: 300, crop: :fill %>
             <% else %>
               <%= image_tag("prep2.jpg", class: 'vote-image',  height: 300, width: 300) %>
+
             <% end %>
 
           </div>
         </div>
         <div class="row justify-content-center">
-          <h4 text-center><strong> <%= @challenge.challenger.title %> </strong></h4>
+
+          <h2 text-center><%= @challenger.title %></h2>
           <div class="row justify-content-center comment-row">
             <div class="d-flex col-12 justify-content-center">
-             <% if @challenge.challenger.caption.nil? %>
-              <p>Waiting for <%= @challenge.challenger.user.username %> to upload...</p>
+             <% if @challenger.caption.nil? %>
+              <p>Waiting for <%= @challenger.user.username %> to upload...</p>
              <% else %>
-              <p><%= @challenge.challenger.caption %></p>
+              <p><%= @challenger.caption %></p>
              <% end %>
             </div>
           </div>
         </div>
       </div>
+
       <% if current_user != @challenge.challenger.user && current_user != @challenge.opponent.user %>
-        <div class="col d-flex justify-content-center mt-3 pt-3 checkmark-row">
-          <div class="col-2 checkmark-icon">
-            <i class="far fa-check-circle"></i>
-          </div>
+      <div class="col d-flex justify-content-center mt-3 pt-3 checkmark-row">
+        <div class="col-2 checkmark-icon">
+           <input type='radio' id='challenger-vote' checked='' name='radio'>
+          <label for='challenger-vote'><i class="far fa-check-circle"></i></label>
         </div>
+        <div class="btn btn-purple" id="ninja-submit2" style="display: none;">
+        Submit Vote
+        </div>
+      </div>
       <% end %>
     </div><!-- /page-user-one -->
 
-    <!-- PAGE USER TWO -->
+    <!-- PA</div>GE USER TWO -->
```

Comment:
no fucking clue what's happening here but it looks like a div close tag got accidentally misplaced? worth checking that nothing broke / that it wasn't missed